packages/blade/android/app/src/main/java/com/blade/MainApplication.java

```diff
@@ -10,6 +10,7 @@
 import com.facebook.soloader.SoLoader;
 import java.lang.reflect.InvocationTargetException;
 import java.util.List;
+import com.facebook.react.views.text.ReactFontManager;
```

Comment:
Move this right below `import com.facebook.react.ReactPackage` to maintain the order?

Reply:
thought not to pay much attention to that part 😢